tests/test_compute.py

```diff
@@ -184,6 +184,13 @@ def sacct_callback(request, uri, response_headers):
             "task_url": "https://148.187.97.214:8443/tasks/acct_full_id",
         }
         status_code = 200
+    elif jobs == ['empty']:
+        ret = {
+            "success": "Task created",
+            "task_id": "acct_empty_id",
+            "task_url": "https://148.187.97.214:8443/tasks/acct_empty_id",
```

Comment:
Why is there a fixed IP address in the response?

Reply:
Good point, I will update it in all the responses